src/tutorials.md

```diff
@@ -0,0 +1,13 @@
+# Tutorials
+
+Welcome! This page contains links to general tutorials useful to astronomers and astrophysicists getting started with Julia.
+
+In Julia, smaller focussed packages are preferred. This page shows how you can combine smaller packages to get big results!
+
+For more advanced usage examples, please see the documentation pages of the individual packages linked on the [Ecosystem](@ref eco) page.
+
+
+
+## General
+
+* [Plot a JWST image with world coordinates and a scale bar](@ref tutorial-jwst-scalebar)
```

Comment:
```suggestion
# Tutorials

The tutorials section contains links to general tutorials useful to astronomers and astrophysicists getting started with Julia.

In Julia, smaller focused packages are preferred over large monolithic packages. It may be a little overwhelming at first to know how to connect them together to enable your science, but we're here to help! You'll find in the tutorials how you can combine smaller packages to get big results!

For more advanced usage examples, please see the documentation pages of the individual packages linked on the [Ecosystem](@ref eco) page.

## General

* [Plot a JWST image with world coordinates and a scale bar](@ref tutorial-jwst-scalebar)
```